### DISCUSSION TITLE
Unable to remove card fields

### DISCUSSION THREAD

**Discussion by SVG4365** [microsoft-search/pnp-modern-search#1670] [Q&A] [answered]
Hi,
I want to remove some fields from the card layout. However the 'delete the current row' isn't working for me. 
What am I doing wrong? (v 4.4.1)

![image](https://user-images.githubusercontent.com/57793572/147075091-23c8407a-a8c5-4ed1-ae6f-afd1c1ec084e.png)

**Comment by FranckyC** [comment, marked as answer]
Hi @SVG4365, this is the expected behavior. You can't remove fields for the card layout. If you don't want the field to appear, just set an empty value in the Handlebars expression.

**Reply by SVG4365** [reply]
Thanks @FranckyC, works fine for me
